<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Hist2DDialog</class>
 <widget class="QDialog" name="Hist2DDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>188</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hist2D</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="le_name"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>X Resolution</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Y Resolution</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>X Source</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="label_xSource">
       <property name="minimumSize">
        <size>
         <width>185</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_xSource">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/tree-settings.png</normaloff>:/tree-settings.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_xClear">
       <property name="toolTip">
        <string>Clear selection</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/dialog-close.png</normaloff>:/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Y Source</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0,0">
     <item>
      <widget class="QLabel" name="label_ySource">
       <property name="minimumSize">
        <size>
         <width>185</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_ySource">
       <property name="text">
        <string>Select</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/tree-settings.png</normaloff>:/tree-settings.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pb_yClear">
       <property name="toolTip">
        <string>Clear selection</string>
       </property>
       <property name="icon">
        <iconset resource="resources.qrc">
         <normaloff>:/dialog-close.png</normaloff>:/dialog-close.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QComboBox" name="comboXResolution"/>
   </item>
   <item row="2" column="1">
    <widget class="QComboBox" name="comboYResolution"/>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Hist2DDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Hist2DDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>216</x>
     <y>205</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>221</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
